spec/models/character_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe Character do
+  Stat.create!(name: "Speed")
+
+  let(:character) { Character.create!(name: "Professor Longfellow") }
+  let(:character_stat) { CharacterStat.create!(character_id: character.id,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.